src/careamics/config/support/supported_losses.py

```diff
@@ -21,8 +21,9 @@ class SupportedLoss(str, BaseEnum):
     MAE = "mae"
     N2V = "n2v"
     # PN2V = "pn2v"
-    # HDN = "hdn"
+    HDN = "hdn"
     MUSPLIT = "musplit"
+    MICROSPLIT = "microsplit"
     DENOISPLIT = "denoisplit"
     DENOISPLIT_MUSPLIT = "denoisplit_musplit"
```

Comment:
I would rename `denoisplit_musplit` into `denoisplit_microsplit` everywhere for consistency. There are a bunch of them laying around.

```suggestion
    DENOISPLIT_MICROSPLIT = "denoisplit_microsplit"
```

Reply:
Should be removed